examples/Cassie/run_osc_walking_controller.cc

```diff
@@ -117,6 +119,11 @@ int DoMain(int argc, char* argv[]) {
   builder.Connect(command_sender->get_output_port(0),
                   command_pub->get_input_port());
 
+  // Create osc debug sender.
+  auto osc_debug_pub =
```

Comment:
Only create the system if it's going to be connected (if the flag is set). Can probably do both in the same if statement.

Reply:
done